WBooksTests/MockBookRepository.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  MockBookRepository.swift
+//  WBooksTests
+//
+//  Created by Juan Silva on 28/06/2021.
+//
+
+import XCTest
+@testable import WBooks
+
+class MockBookRepository: BookRepositoryProtocol {
+    private var bookJSON: String!
```

Comment:
If this property will not change its value, you can declare it as `let` instead of `var`. Also, the `!` is not necessary here.